pontoon/tags/utils/resources.py

```diff
@@ -99,16 +97,13 @@ def link(self, tag, glob=None, resources=None):
                     path=resource["path"])
             tag.resources.add(*list(_resources))
 
-    def unlink(self, tag, glob=None, resources=None):
-        """ Unassociate Resources from a Tag. The Resources can be selected
-        either by passing a glob expression to match, or by passing a list
-        of Resource paths
+    def unlink(self, tag, resources):
+        """
+        Unassociate Resources from a Tag.
+
+        The Resources can be selected by passing a list of Resource paths.
         """
         query = Q(project__in=self.projects) if self.projects else Q()
-        if glob is not None:
```

Comment:
Support for glob expressions serves here as a helper in unit-tests. IMHO this method doesn't need this additional complexity.